staging/src/k8s.io/api/resource/v1beta1/types.go

```diff
@@ -436,10 +436,11 @@ type DeviceRequest struct {
 	//   count field.
 	//
 	// - All: This request is for all of the matching devices in a pool.
+	//   At least one device must be available on the node for the allocation to succeed.
 	//   Allocation will fail if some devices are already allocated,
 	//   unless adminAccess is requested.
 	//
-	// If AlloctionMode is not specified, the default mode is ExactCount. If
+	// If AllocationMode is not specified, the default mode is ExactCount. If
```

Comment:
@pohly fixed a typo along the way. let me know if you prefer this to be in a separate commit.

Reply:
Single commit is fine.

Reply:
Please update the internal types.go and v1alpha3 the same way.

You also have to re-generate several files.

Reply:
done